lib/fathead/vuejs/output.txt

```diff
@@ -0,0 +1,88 @@
+silent	A			Global Config						https://vuejs.org/images/logo.png	<section class="prog__container"><pre><code>silent</code></pre><p><span class="prog__sub">Type:</span> <code>boolean</code></p>\n<p><span class="prog__sub">Default:</span> <code>false</code></p>\n<p><span class="prog__sub">Usage:</span></p><p>Suppress all Vue logs and warnings.</p><pre><code>Vue.config.silent = true\n</code></pre></section>	https://vuejs.org/v2/api/#silent
```

Comment:
nit: It looks like we're wrapping the `<span class="prog__sub">` in a `<p>` -- that's not necessary, just add extra spacing.